README.md

```diff
@@ -406,7 +406,7 @@ To generalize string data type using expression to define the value set mapping
 
 ```json
 {
-  "path": "patient.communication.language",
+  "path": "Patient.communication.language.coding",

```

Comment:
Seems to be Patient.communication.language.coding.code?

Reply:
Fixed it, thanks.